docs/_basic/responding_actions.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Responding to actions
+slug: action-respond
+order: 8
+---
+
+<div class="section-content">
+There are two main ways to respond to actions. The first way (and the most common way) is using the `say` function. The `say` function sends a message back to the conversation where the incoming event took place.
+
+The second way to respond to actions is using `respond()`. This is a simple utility to use the `response_url` associated with an action.
+</div>
+
+```javascript
+// Your middleware will be called every time an interactive component with the action_id “approve_button” is triggered
+app.action('approve_button', async ({ ack, say }) => {
+  ack();
+  say('Request approved 👍');
+});
+```
+
+<details class="secondary-wrapper">
+<summary class="section-head" markdown="0">
+<h4 class="section-head">Using respond()</h4>
+</summary>
+
+<div class="secondary-content" markdown="0">
+In addition to using `say()`, you can respond to them using `respond()`. `respond()` is a utility for calling the `response_url`, so it behaves in the same way. You can pass a JSON object with a new message payload that will be published back to the source of the original interaction. Response URLs are covered in more depth [on our API site](https://api.slack.com/messaging/interactivity).
```

Comment:
"it behaves in the same way" doesn't feel like its saying anything (same as what? i know you mean the `response_url` but as i'm reading i'm expecting you to tell me why i would use this instead of `say()` and this makes it seem like there is no difference). i think the second sentence needs to describe the difference. i suggest that you specifically mention `response_type`, `replace_original`, and/or `delete_original` - or at least allude to where these details can be found.